### PR TITLE
Move from MYSQL_DISTRIBUTION -> DB_DISTRIBUTION

### PR DESCRIPTION
**Pull request by navarr**
Part of ensuring bc-compatibility with Den

### CODE REVIEW

File: environments/magento1/init.env [hunk 1]
```diff
@@ -2,8 +2,8 @@
 WARDEN_DB=1
 WARDEN_REDIS=1
 
-MYSQL_DISTRIBUTION=mariadb
-MYSQL_DISTRIBUTION_VERSION=10.3
+DB=mariadb
```
Comment (bap14) [inline environments/magento1/init.env:5]: @navarr Is this missing the `_DISTRIBUTION` part of the key or is this a weird edge case?

Comment (navarr) [reply environments/magento1/init.env:5]: Good catch.  Appreciate you reviewing my WIPs.  <3

File: environments/magento1/init.env [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
+DB_VERSION=10.3
```
Comment (navarr) [inline environments/magento1/init.env:6]: `
```suggestion
DB_DISTRIBUTION=mariadb
DB_DISTRIBUTION_VERSION=10.3
```

File: commands/env.cmd [hunk 3]
```diff
@@ -9,6 +9,9 @@ if (( ${#WARDEN_PARAMS[@]} == 0 )) || [[ "${WARDEN_PARAMS[0]}" == "help" ]]; the
   $WARDEN_BIN env --help || exit $? && exit $?
 fi
 
+DB_DISTRIBUTION=${DB_DISTRIBUTION:-${MYSQL_DISTRIBUTION:-"mariadb"}}
+DB_DISTRIBUTION_VERSION=${DB_DISTRIBUTION_VERSION:-${MYSQL_DISTRIBUTION_VERSION:-${MARIADB_VERSION:-"10.4"}}}
```
Comment (ihor-sviziev) [inline commands/env.cmd:13]: 1. shoudln't we add `export` there?
```suggestion
export DB_DISTRIBUTION=${DB_DISTRIBUTION:-${MYSQL_DISTRIBUTION:-"mariadb"}}
export DB_DISTRIBUTION_VERSION=${DB_DISTRIBUTION_VERSION:-${MYSQL_DISTRIBUTION_VERSION:-${MARIADB_VERSION:-"10.4"}}}
```

2. maybe it's better to pring some warning if the DB_DISTRIBUTION and DB_DISTRIBUTION_VERSION variable aren't declared yet? In this case we might suggest changing MYSQL_DISTRIBUTION variable to DB_DISTRIBUTION and MYSQL_DISTRIBUTION_VERSION to DB_DISTRIBUTION_VERSION. What do you think?

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: commands/env.cmd (hunk 3 above)
Comment (navarr) [reply commands/env.cmd]: I don't exactly follow #2.

Warden's old default is MariaDB 10.4.  A few versions ago I mistakenly introduced the ability to change database distribution and version using MYSQL_ variables (the mistake)

Moving to DB_DISTRIBUTION so it matches Den is what I'm trying to do here, while maintaining backwards compatibility with any `.env` file that uses MYSQL_ variables, and maintaining backwards compatibility with the original defaults

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
@navarr just wonder, why you closed it?

**Comment by navarr** [comment]
@ihor-sviziev it doesn't feel like there's a point in the backwards compatibility after all this time

**Pull request closed without merging**